src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import { StyleSheet, View, Text, Image } from "react-native";
+
+import { styled } from "nativewind";
+
+import { AvatarType } from "./Avatar.types";
+import { getAvatarColor, getAvatarInitials } from "../../utils";
+
+const Avatar: React.FC<AvatarType> = ({ username, image }) => {
+	const [backgroundColor, setBackgroundColor] = React.useState("");
+
+	let avatarStyle = { ...style.avatarBase, backgroundColor };
```

Comment:
you might want to look into using compose here. Let's not mix styles like this anymore, let's try to focus on using StyleSheet.compose